### PR TITLE
Adobe analytics implementation

### PR DESCRIPTION
**Pull request by arnab-p**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by saikumarrs — DISMISSED** [review]

File: versionedRouter.js [hunk 1]
```diff
@@ -619,6 +619,7 @@ router.post("/batch", ctx => {
   batchHandler(ctx);
 });
 
+
```
Comment (saikumarrs) [inline versionedRouter.js:622]: Please revert this.

Comment (shrouti1507) [reply versionedRouter.js]: reverted

**Review by utsabc — APPROVED** [review]

File: destTransformer.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const { testRouter } = require("./testRouter");
 const cluster = require("./util/cluster");
 const { addPrometheusMiddleware } = require("./middleware");
 
-const clusterEnabled = true;
+const clusterEnabled = process.env.CLUSTER_ENABLED === false ? false : true ;
```
[the remark this answers is not in the document]
Comment (saikumarrs) [reply destTransformer.js:11]: I'm not sure but the original implementation might have tried to safeguard against non-boolean values.
Perhaps, we could do something like below,

```
!(process.env.CLUSTER_ENABLED === false);
```

[the remark this answers is not in the document]
Comment (utsabc) [reply destTransformer.js:11]: @shrouti1507 lets address this

[the remark this answers is not in the document]
Comment (shrouti1507) [reply destTransformer.js]: done

**Review by saikumarrs — APPROVED** [review]

File: destTransformer.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ const { testRouter } = require("./testRouter");
 const cluster = require("./util/cluster");
 const { addPrometheusMiddleware } = require("./middleware");
 
-const clusterEnabled = true;
+const clusterEnabled = !(process.env.CLUSTER_ENABLED === false) || true;
```
Comment (akashrpo) [inline destTransformer.js:11]: This will return `true` regardless of `CLUSTER_ENABLED` value

**Pull request merged**